### PR TITLE
Chore: ckbtc scripts deploy to testnet

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Use ckbtc scripts to deploy ckbtc canisters to a testnet.

# Changes

* Change dfx.json with where the ckbtc related wasms and did files live once downloaded with script.
* Rename the canisters in the ckbtc file to match the ones in dfx.json

# Tests

Not applicable

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @bitdivine please review

**Comment by peterpeterparker** [comment]
Isn't this a duplicate of the task - already reviewed but not merged - of @bitdivine https://github.com/dfinity/nns-dapp/pull/1905/files?

**Comment by lmuntaner** [comment]
> Isn't this a duplicate of the task - already reviewed but not merged - of @bitdivine https://github.com/dfinity/nns-dapp/pull/1905/files?

If the PR used the scripts in the ckbtc directory yes. But they deploy directly with `dfx`.

Not sure whether we could use `deploy-snsdemo-testnet` directly. But that seems more a script that deploys all canisters than the individual ones.

But it's tighly related to deploying the ckbtc. I fear there is some duplication that we should fix to test them both.

File: dfx.json [hunk 1]
```diff
@@ -109,21 +109,29 @@
...
+      "type": "custom"
+    },
     "ckbtc_index": {
       "build": [
         "true"
       ],
-      "candid": "target/ic/ic-icrc1-index.did",
-      "wasm": "target/ic/ic-icrc1-index.wasm",
+      "candid": "tmp/index.did",
+      "wasm": "tmp/index_canister.wasm",
```
Comment (bitdivine) [inline dfx.json:134]: Is the above change intended?  It looks to me as if the path on the left is correct, no?

**Review by bitdivine — APPROVED** [review]
Looks good.  There is one path that I am unsure of; please see the comment.

File: dfx.json (hunk 1 above)
Comment (lmuntaner) [reply dfx.json:134]: This is where the download scripts puts them:

```
$ ./scripts/ckbtc/download-ckbtc
tmp/ckbtc_minter.wasm already exists -- do you wish to overwrite (y or n)? y
tmp/ledger_canister.wasm already exists -- do you wish to overwrite (y or n)? y
tmp/index_canister.wasm already exists -- do you wish to overwrite (y or n)? y
```

We can improve this later?

Comment (bitdivine) [reply dfx.json:134]: Sure, I didn't see the download scripts.  No need to change.

Comment (lmuntaner) [reply dfx.json]: I changed the download script 😂 

I like consistency.

**Pull request merged**